Todo_CLI/homework_3.js

```diff
@@ -0,0 +1,126 @@
+const readline = require('readline');
+const rl = readline.createInterface({
+    input : process.stdin,
+    output : process.stdout,
+    prompt : "(v) View • ( n ) New • (cX) Complete • (dX) Delete • (s) Save • (q) Quit"+'\n'+">"
+});
+
+let tasks= [];
+
+
+const result=()=> {
```

Comment:
common practice is that you should define the functions that will be used in `result` above this.